test/pull_data.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+
+use PHPunit\Framework\Testcase;
+
+require '../pull_data.php';
```

Comment:
should be requiring your functions.php file, not this one. this breaks your unit tests